### PR TITLE
Update experiments-msmarco-passage.md

### PR DESCRIPTION
**Pull request by gituserbs**
Setup Details  (different form my previous setup):
Operating System: [Ubuntu 18.04.6]
Environment and Configuration: [Python 3.10.9, Git version: 2.17.1, Java 11] 
Additional Setup Information: [Intel® Xeon(R) Gold 5118 CPU @ 2.30GHz × 48, RAM: 128 GB] 
Outcome: [Success, all the steps worked successfully.]

### CODE REVIEW

**Comment by lintool** [comment]
Please continue along the onboarding path until you reach the end of the exercises, and then we'll take a look all at once.

**Comment by lintool** [comment]
@gituserbs did you do https://github.com/castorini/anserini/blob/master/docs/start-here.md ?

**Comment by gituserbs** [comment]
Yes, I have completed that step. The work was initially part of a different pull request (#2218  ), which has recently been closed. I have since consolidated the changes from that pull request into the current one.

**Comment by lintool** [comment]
Please order your log entry chronologically.

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -439,4 +439,6 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@dourian](https://github.com/dourian) on 2023-09-25 (commit [`24ab292`](https://github.com/castorini/anserini/commit/24ab292c5eaaccd40bbfa13fa7122eeb58261aaa))
 + Results reproduced by [@ksunisth](https://github.com/ksunisth) on 2023-09-27 (commit [`7c95d91`](https://github.com/castorini/anserini/commit/7c95d9155e59cfc55c573139b3538cc76d0f5371))
 + Results reproduced by [@maizerrr](https://github.com/maizerrr) on 2023-10-01 (commit [`2abdd37`](https://github.com/castorini/anserini/commit/2abdd373d70ae64ff6638c3440d7f0063514d313))
++ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-14 (commit [`8194b8e`](https://github.com/castorini/anserini/commit/8194b8e91fe799fcb0d3b4348e9d710a0aa1ad26))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:442]: move down, chronologically

File: docs/start-here.md [hunk 2]
```diff
@@ -334,3 +334,4 @@ If you think this guide can be improved in any way (e.g., you caught a typo or t
 + Results reproduced by [@ksunisth](https://github.com/ksunisth) on 2023-09-27 (commit [`7c95d91`](https://github.com/castorini/anserini/commit/7c95d9155e59cfc55c573139b3538cc76d0f5371))
 + Results reproduced by [@maizerrr](https://github.com/maizerrr) on 2023-10-01 (commit [`2abdd37`](https://github.com/castorini/anserini/commit/2abdd373d70ae64ff6638c3440d7f0063514d313))
 + Results reproduced by [@shayanbali](https://github.com/shayanbali) on 2023-10-12 (commit [`8194b8e`](https://github.com/castorini/anserini/commit/8194b8e91fe799fcb0d3b4348e9d710a0aa1ad26))
++ Results reproduced by [@gituserbs](https://github.com/gituserbs) on 2023-10-07 (commit [`d88003c`](https://github.com/castorini/anserini/commit/d88003c71ae7b2e3da65885c709f61aac57611b4))
```
Comment (lintool) [inline docs/start-here.md:337]: move up, chronologically

File: docs/experiments-msmarco-passage.md (hunk 1 above)
Comment (gituserbs) [reply docs/experiments-msmarco-passage.md]: Adjusted the order of log entries to follow a chronological sequence as requested. Apologies for the oversight.

File: docs/start-here.md (hunk 2 above)
Comment (gituserbs) [reply docs/start-here.md:336]: Adjusted the order of log entries to follow a chronological sequence as requested. Apologies for the oversight.

**Review by lintool — APPROVED** [review]

**Pull request merged**